composer.json

```diff
@@ -15,7 +15,8 @@
     "require": {
         "php": ">=5.3.3",
```

Comment:
So do you want me to remove the usage of ::class and use FQDN strings instead? or update composer.json to require PHP 5.5 instead?

I prefer the later as Zend\ServiceManager has required PHP 5.5 since 2.6